project/ProjectPlugin.scala

```diff
@@ -22,7 +22,7 @@ object ProjectPlugin extends AutoPlugin {
     lazy val V = new {
       val catsEffect: String    = "2.1.3"
       val circe: String         = "0.13.0"
-      val frees: String         = "0.8.2"
+      val doobie: String        = "0.8.2"
```

Comment:
It doesn't really matter but the latest doobie is 0.9.0

Reply:
couldn't hurt to upgrade!